sandbox-templates/remark-with-parcel/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "remark-with-parcel",
+  "version": "1.0.0",
+  "private": true,
+  "main": "index.html",
+  "scripts": {
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html"
+  },
+  "dependencies": {
+    "remark": "14.0.2"
+  },
+  "devDependencies": {
+    "@babel/core": "7.16.0",
+    "parcel-bundler": "1.12.5"
+  }
+}
```

Comment:
```suggestion
}
```

Reply:
Something seems to have gotten messed up with the suggestion, it appears to suggest not changing the code?

Reply:
I think a newline was intended? added